Schemas/configuration.json

```diff
@@ -27,7 +27,9 @@
                     "exp_image_downloader_priority",
                     "exp_text_drawing",
                     "exp_fix_range_controller",
-                    "exp_oom_bg_dealloc_disable"
+                    "exp_oom_bg_dealloc_disable",
+                    "exp_transaction_operation_retain_cycle",
+                    "exp_adding_lock_to_textkit_initialising"
```

Comment:
The experiment should remove the lock so that current behavior is maintained ("opt-in".)